src/sgame/sg_bot_nav.cpp

```diff
@@ -689,56 +690,69 @@ void BotMoveToGoal( gentity_t *self )
 		targetTeam = G_Team( self->botMind->goal.getTargetedEntity() );
 	}
 
-	usercmd_t &botCmdBuffer = self->botMind->cmdBuffer;
-	switch ( self->client->pers.team )
+	// if target is friendly, let's go there quickly (heal, poison) by using trick moves
+	// when available (still need to implement wall walking, but that will be more complex)
+	if ( G_Team( self ) != targetTeam )
 	{
-		case TEAM_HUMANS:
-			// if target is friendly, reach it quickly
-			if ( G_Team( self ) == targetTeam )
+		return;
+	}
+
+	usercmd_t &botCmdBuffer = self->botMind->cmdBuffer;
+	vec3_t dest;
```

Comment:
This variable is unnecessary

Reply:
It is, because vec3_t can not be easily copied on stack (passing byval) and CalcAimPitch modifies it (it would be better with a byval).

Reply:
In any cases, I moved the declaration closer to where it's used.